CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.2-dev
+
+* Update `pipeline_api` to support `text/csv` response
+* Add new function `convert_to_isd_csv`
+* Add test function `test_section_narrative_api_with_text_csv_response_type`
```

Comment:
Apologies for not being more specific on the CHANGELOG update. Just one bullet mentioning CSV support is needed, added underneath `0.2.1-dev` (at line 9, pushing existing line down) is sufficient.

When it is just a `-dev update`, a new version is not needed.

Reply:
Just added `Support text/csv response type` underneath `0.2.1-dev`